components/chef-ui-library/package-lock.json

```diff
@@ -8107,4 +8107,4 @@
       }
     }
   }
-}
+}
```

Comment:
Add empty line at end of page

Reply:
@sumanprogress please take care of this next time, the same comment I have posted on your other PR.

Reply:
new line added